b2share/modules/records/serializers/__init__.py

```diff
@@ -51,5 +50,13 @@
 oaipmh_oai_dc = dc_v1.serialize_oaipmh
 oaipmh_marc21_v1 = marcxml_v1.serialize_oaipmh
 
+class DataCite31Serializer(InvenioDataCite31Serializer):
+    """ Subclassing invenio DataCite31Serializer to overrule method marshmallow dump """
+
+    def dump(self, obj, context=None):
+        """Serialize object with schema."""
+        return self.schema_class(context=context).dump(obj)
```

Comment:
dump method is no longer existing in the invenio package. This workaround is needed since our code still uses this method. 
Double check if this really works since this is because of marshmallow changes -HH